discord_script.py

```diff
@@ -1,7 +1,8 @@
+import sys
 import random
 import discord
 import scraping_google_images
-import database_creation
+# import database_creation
```

Comment:
Why is this commented out?

Reply:
This comment is needed to prevent the pipeline from failing due to errors with the database.